dali/base/dasds.cpp

```diff
@@ -5236,7 +5236,7 @@ class CStoreHelper : public CInterface, implements IStoreHelper
                 MilliSleep(100);
             }
         }
-        ~CheckDeltaBlock()
+        ~CheckDeltaBlock() noexcept(false)
```

Comment:
The whole purpose of CheckDeltaBlock (as far as I can tell) is to throw the exception if the check fails, and to ensure that the check is done.